phpBB/phpbb/textformatter/s9e/factory.php

```diff
@@ -84,6 +84,12 @@ class factory implements \phpbb\textformatter\cache_interface
 		'img'   => '[IMG src={IMAGEURL;useContent}]',
 		'list'  => '[LIST type={HASHMAP=1:decimal,a:lower-alpha,A:upper-alpha,i:lower-roman,I:upper-roman;optional;postFilter=#simpletext} #createChild=LI]{TEXT}[/LIST]',
 		'li'    => '[* $tagName=LI]{TEXT}[/*]',
+		'mention' =>
+			"[MENTION
+				group_id={UINT;optional}
```

Comment:
It might be better to merge all `*_id` tags into one and have that information in the `id` tag. E.g. instead of this: `group_id=foobar` it could be `id=group-foobar`.